src/components/Swap/index.tsx

```diff
@@ -61,6 +62,7 @@ export default function Swap(props: SwapProps) {
   const isDisabled = !(isActive && onSupportedNetwork)
 
   const focused = useHasFocus(wrapper)
+  const disableBranding = useBrandingSetting()
```

Comment:
nit: `useBrandingSetting` should be renamed so it's output is more obvious. I'd be in favor of just returning `BrandingSettings`, and then destructuring.